check-breaking/action.yml

```diff
@@ -1,20 +1,25 @@
-name: 'OpenAPI Spec: check for breaking API changes'
-description: 'Detect breaking changes in OpenAPI Specification 3'
+name: "OpenAPI Spec: check for breaking API changes"
```

Comment:
The `"` to '`' changes were made by my editor automatically. I can revert them back if you prefer using `'`